surface.js

```diff
@@ -200,6 +205,22 @@ proto.setPickBase = function (id) {
   this.pickId = id
 }
 
+function getOpacityFromScale(ratio, opacityscale) { // copied form gl-mesh3d
```

Comment:
It would be nice to start a helper module in gl-vis to reuse logic like this in both gl-mesh3d and gl-surface3d.